src/test/regress/expected/custom_aggregate_support.out

```diff
@@ -119,29 +119,29 @@ GROUP BY(1);
 ------------------------------------------------------------------------
  HashAggregate
    Group Key: remote_scan.day
-   ->  Custom Scan (Citus Real-Time)
+   ->  Custom Scan (Citus Adaptive)
          Task Count: 4
          Tasks Shown: All
          ->  Task
                Node: host=localhost port=57637 dbname=regression
                ->  HashAggregate
                      Group Key: day
-                     ->  Seq Scan on daily_uniques_360289 daily_uniques
+                     ->  Seq Scan on daily_uniques_360453 daily_uniques
```

Comment:
Would be a good moment to add `SET citus.next_shard_id TO ...;` at the top of the test to make shard identifiers stable.